cle/backends/elf/elf.py

```diff
@@ -866,10 +834,52 @@ def __relocate_mips(self):
             self.relocs.append(reloc)
 
         for i in range(symbol_count - symtab_got_idx):
-            symbol = self.get_symbol(i + symtab_got_idx)
+            symbol = self.get_symbol(i + symtab_got_idx, symtab)
             reloc = MipsGlobalReloc(self, symbol, gotaddr + (i + got_local_num)*wordsize)
             self.relocs.append(reloc)
             self.jmprel[symbol.name] = reloc
         return True
 
+    def __neuter_streams(self, obj):
+        if isinstance(obj, elftools.elf.dynamic._DynamicStringTable):
+           obj._stream = self.memory
+           obj._table_offset = self._offset_to_rva(obj._table_offset)
+        elif isinstance(obj, elftools.elf.sections.Section):
+            obj.stream = self.memory
+            obj.elffile = None
+            obj.header.sh_offset = self._offset_to_rva(obj.header.sh_offset)
+        else:
+            raise TypeError("Can't convert %r" % type(obj))
+
+    def _offset_to_rva(self, offset):
+        return AT.from_mva(self.offset_to_addr(offset), self).to_rva()
+
+    def __process_debug_file(self, filename):
+        with open(filename, 'rb') as fp:
+            try:
+                elf = elffile.ELFFile(fp)
+            except ELFError:
+                l.warning("pyelftools failed to load debug file %s", filename, exc_info=True)
+                return
+
+            for sec_readelf in elf.iter_sections():
+                if isinstance(sec_readelf, elffile.SymbolTableSection):
+                    self.__register_section_symbols(sec_readelf)
+                elif sec_readelf.header.sh_type == 'SHT_NOTE':
+                    self.__register_notes(sec_readelf)
+
+            #has_dwarf_info = bool(elf.has_dwarf_info())
+            #if has_dwarf_info:
+            #    try:
+            #        dwarf = self.reader.get_dwarf_info()
+            #    except ELFError:
+            #        l.warning("An exception occurred in pyelftools when loading the DWARF information on %s.", filename,
+            #                  exc_info=True)
+            #        dwarf = None
+
+                # debug symbols don't have eh_frame ever from what I can tell
+                #if dwarf and dwarf.has_EH_CFI():
+                #    self._load_function_hints_from_fde(dwarf, FunctionHintSource.EXTERNAL_EH_FRAME)
```

Comment:
I suppose the above commented lines are useless and can be removed?

Reply:
I wanna leave these because there will be a day when we want to extract dwarf information from these files, and when that day comes we will want the reminder that debug files don't have eh_frame in them.